### PR TITLE
fix count of days in countdowns

### PR DESCRIPTION
**Pull request by dkent600**
Resolves: https://github.com/daostack/alchemy/issues/1763

### CODE REVIEW

File: src/components/Shared/ProposalCountdown.tsx [hunk 1]
```diff
@@ -50,7 +49,7 @@ export default class ProposalCountdown extends React.Component<IProps, IState> {
     this.stop();
   }
 
-  public calculateCountdown(endDate: Date | moment.Moment) {
+  private calculateCountdown(endDate: Date | moment.Moment) {
```
Comment (orenyodfat) [inline src/components/Shared/ProposalCountdown.tsx:52]: How about using a single calculateCountdown  util function ?

Comment (dkent600) [reply src/components/Shared/ProposalCountdown.tsx:52]: Done

Comment (orenyodfat) [reply src/components/Shared/ProposalCountdown.tsx:52]: there is no commit

Comment (dkent600) [reply src/components/Shared/ProposalCountdown.tsx]: check again

Comment (dkent600) [reply src/components/Shared/ProposalCountdown.tsx]: https://github.com/daostack/alchemy/pull/1764/commits/e2cacc1f5d18bc16e3d264819e17f38f56374cb5

**Pull request merged**